docs/v2.4/reference/rbac.md

```diff
@@ -41,7 +41,79 @@ After creating `config.yaml` in your current directory, export the file's locati
 
 ## Administrating Telepresence
 
-[Telepresence administration](/products/telepresence/) requires permissions for creating `Namespaces`, `ServiceAccounts`, `ClusterRoles`, `ClusterRoleBindings`, `Secrets`, `Services`, `MutatingWebhookConfiguration`, and for creating the `traffic-manager` [deployment](../architecture/#traffic-manager) which is typically done by a full cluster administrator.
+Telepresence administration requires permissions for creating `Namespaces`, `ServiceAccounts`, `ClusterRoles`, `ClusterRoleBindings`, `Secrets`, `Services`, `MutatingWebhookConfiguration`, and for creating the `traffic-manager` [deployment](../architecture/#traffic-manager) which is typically done by a full cluster administrator. The following permissions are needed for the instillation and use of Telepresence.
+
+```yaml
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: telepresence-test-developer
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: telepresence-role
+rules:
+  - apiGroups:
+      - ""
+    resources: ["pods", "pods/log"]
+    verbs: ["get", "list", "create", "delete"]
+  - apiGroups:
+      - ""
+    resources: ["services"]
+    verbs: ["get", "list", "update", "create"]
+  - apiGroups:
+      - ""
+    resources: ["pods/portforward"]
+    verbs: ["create"]
+  - apiGroups:
+      - "apps"
+    resources: ["deployments", "replicasets", "statefulsets"]
+    verbs: ["get", "list", "update"]
+  - apiGroups:
+      - "getambassador.io"
+    resources: ["hosts", "mappings"]
+    verbs: ["*"]
+  - apiGroups:
+      - ""
+    resources: ["endpoints"]
+    verbs: ["get", "list"]
+  - apiGroups:
+      - "rbac.authorization.k8s.io"
+    resources: ["clusterroles", "clusterrolebindings", "roles", "rolebindings"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups:
+      - ""
+    resources: ["namespaces"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups:
+      - ""
+    resources: ["secrets"]
+    verbs: ["get", "create"]
+  - apiGroups:
+      - ""
+    resources: ["serviceaccounts"]
+    verbs: ["get", "create"]
+  - apiGroups:
+      - "admissionregistration.k8s.io"
+    resources: ["mutatingwebhookconfigurations"]
+    verbs: ["get"]
```

Comment:
We create a MutatingWebhookConfiguration when a user installs Telepresence for the first time, so I think you need to include create here as well.

Reply:
I see, if it is not deleted on uninstall -e, I might have missed it, I will do tests on fresh clusters